modules/filters.html

```diff
@@ -0,0 +1,153 @@
+<!DOCTYPE html>
+<html lang="en-us">
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width" />
+  <title></title>
+  <style>
+/* To see the value of the current `filter` property in effect, inspect the image in your developer tools. */ 
+article {
+  display: grid; 
+  grid-template-columns: minmax(200px, 800px) 20em;
+  gap: 1rem;
+  max-width: 100%;
+}
+label {
+  display: block; 
+  font-family: sans-serif;
+}
+input {
+  vertical-align: middle; 
+  margin-right: 0.25em; 
+  max-width: 50%;
+}
+img {
+  margin: 1rem;
+  width: 100%;
+  object-fit: cover;
+  overflow: hidden;
+}
+  </style>
+</head>
+<body>
+<article>
+<img src="assets/george_floyd_memorial_sm.jpg" width="600" height="400" alt="A colorful memorial honoring George Floyd and the Black Lives Matter movement"/>
+<form>
+<fieldset>
+  <legend>Select your filters</legend>
+    <label>
+      <input type="range" name="blur" value="0" min="0" max="1" step="0.1">
+      blur()
+    </label>
+    <label>
+      <input type="range" name="brightness" value="1" min="0" max="2" step="0.1">
+      brightness()
+    </label>
+    <label>
+      <input type="range" name="contrast"  value="1" min="0" max="2" step="0.1">
+      contrast()
+    </label>
+    <label>
+      <input type="range" name="dropShadow"  value="0" min="-1" max="1" step="0.1">drop-shadow()
+    </label>
+    <label>
+      <input type="range" name="grayscale"  value="0" min="0" max="1" step="0.1">
+      grayscale()
+    </label>
+    <label>
+      <input type="range" name="hueRotate"  value="0" min="-1" max="1" step="0.1">
+      hue-rotate()
+    </label>
+    <label>
+      <input type="range" name="invert"  value="0" min="0" max="1" step="0.1">
+      invert()
+    </label>
+    <label>
+      <input type="range" name="opacity"  value="1" min="0" max="1" step="0.1">
+      opacity()
+    </label>
+    <label>
+      <input type="range" name="saturate"  value="1" min="0" max="2" step="0.1">
+      saturate()
+    </label>
+    <label>
+      <input type="range" name="sepia"  value="0" min="0" max=" 1" step="0.1">
+      sepia()
+    </label>
+  <button type="reset">Reset</button>
+</fieldset>
+</form>
+</article>
+<script>
+const image = document.querySelector('img');
+const controls = document.querySelectorAll('input');
+let resetValues = new Array();
+
+for(control of controls) {
```

Comment:
Typically, the missing space – it should be `for (` – will be fixed by Prettier.